classes/controller/Controller.php

```diff
@@ -499,6 +499,9 @@ public function addJS($js_uri, $check_path = true)
             $js_path = $js_file = $js_file[0];
             if ($check_path) {
                 $js_path = Media::getJSPath($js_file);
+                if ($js_path === false) {
```

Comment:
It's a BC break, the old method wasn't throwing anything.

You have to change the PHPDoc for this method and add ` * @throw PrestaShopException`